lorax.spec

```diff
@@ -27,7 +27,7 @@ Requires:       dosfstools
 Requires:       e2fsprogs
 Requires:       findutils
 Requires:       gawk
-Requires:       genisoimage
+Requires:       xorrisofs
```

Comment:
This package does not seem to exist in Fedora.

Reply:
Ah yes, this package should be `xorriso`.

Reply:
I don't think it's that simple. The package called `xorriso` was last built in 2013 for Fedora 20 and has been retired since Fedora 22, seemingly (f21 is the last release tag it has).

Reply:
`xorriso` source package is `libisoburn`. It stopped being a separate source package in 2013.

Reply:
Ah, thanks. I was commenting from my phone so couldn't look up package provides :)